assets/sass/patterns/atoms/definition-list.scss

```diff
@@ -3,34 +3,71 @@
 .definition-list--timeline {
   dt,
   dd {
+    @include padding(28, "left");
     font-family: $font-secondary;
     font-weight: bold;
+    position: relative;
   }
 
   dt {
-    @include font-size-and-vertical-height(14, 24);
+    @include font-size-and-vertical-height(16, 24);
     @include margin(18, "top");
+    @include padding(3, "bottom");
 
     &:first-child {
       @include margin(0, "top");
     }
   }
 
   dd {
-    @include font-size-and-vertical-height(12, 24);
+    @include font-size-and-vertical-height(14, 24);
     color: $color-text-secondary;
 
+    .version {
+      color: $color-text;
+
+      &:after {
+        @include padding(0 5 0);
+        content: "\00a0\2219\00a0";
+        color: $color-text-secondary;
+        font-weight: bold;
+      }
+    }
+
     a {
+      @include padding(3, "top");
```

Comment:
Can we remove the padding from the top and add it to the bottom of the <time> element

Reply:
That padding is removed as not needed.